### DISCUSSION TITLE
What was the "latest" Docker image referring to in June 2021

### DISCUSSION THREAD

**Discussion by mherger** [mikefarah/yq#888] [Q&A]
I'm having a problem running the latest yq Docker image in a Github action. I know, there's a specific action for yq. But this is a script which is run locally, or in jenkins, or from a Github action. I _believe_ this used to work until not too long ago. But now I'm getting errors when run on Github (none when run locally on a Mac):

```
05:13:55 safelyRenameFile [ERRO] Failed copying from /tmp/temp093547557 to documentation/api.yaml
05:13:55 safelyRenameFile [ERRO] open documentation/api.yaml: permission denied
``` 

As I believe this did work, I was wondering whether this was due to the latest official release of v4. If I wanted to see whether "latest" from a few weeks back did work, what tag should I use?

**Comment by mikefarah** [comment]
That looks like a bug in 4.9.7 that should have been fixed in 4.9.8 - is that the version you are using?

You can use dockerhub to see the list of versions and the dates they were published: https://hub.docker.com/r/mikefarah/yq/tags?page=1&ordering=last_updated

**Reply by mikefarah** [reply]
(Bug ref: https://github.com/mikefarah/yq/issues/884)

**Reply by mherger** [reply]
Thanks for getting back to me so quickly!

I wasn't specifying a tag, thus was using "latest" - which is a moving target. That's why I was asking what "latest" might have been back when I first created the action.

Would today's "latest" have the bug fix, I assume? Or what other tags should I be using?

**Reply by mherger** [reply]
I'm sorry to say that 4.9.8 still fails. I previously tested 4.6.something successfully. I'll try 4.9.6 next.

**Reply by mherger** [reply]
4.9.6 is working fine - thanks for the pointer!

**Reply by mikefarah** [reply]
Hey @mherger - I thing I've found the root cause for the GitHub action issue - when you get a chance could you try 4.10.0 and let me know how it goes?

**Reply by mherger** [reply]
I'm sorry, 4.10 is still not working:

```
Status: Downloaded newer image for mikefarah/yq:4.10.0
13:22:38 safelyRenameFile [ERRO] Failed copying from /tmp/temp121340422 to documentation/api.yaml
13:22:38 safelyRenameFile [ERRO] open documentation/api.yaml: permission denied
```
